packages/hardhat-etherscan/README.md

```diff
@@ -120,6 +122,56 @@ module.exports = {
 };
 ```
 
+### Multiple API keys and alternative block explorers
+
+If your project targets multiple evm sidechains, you many need to verify using the block explorers particular to those chains, and so need to set multiple API keys.
+
+A block explorer API key can be set per chain, hardhat will use the `chainId` of the specified network to determine which block explorer and API key to use.
+
+To configure the API keys for the chains you are using, provide an object with under `etherscan/apiKey` with properties matching the chain:
+
+```js
+module.exports = {
+  networks: {
+    mainnet: { ... },
+    testnet: { ... }
+  },
+  etherscan: {
+    apiKey: {
+        mainnet: "YOUR_ETHERSCAN_API_KEY",
+        ropsten: "YOUR_ETHERSCAN_API_KEY",
+        rinkeby: "YOUR_ETHERSCAN_API_KEY",
+        goerli: "YOUR_ETHERSCAN_API_KEY",
+        kovan: "YOUR_ETHERSCAN_API_KEY",
+        // binance smart chain
+        bsc = "YOUR_BSCSCAN_API_KEY",
+        bsc_testnet = "YOUR_BSCSCAN_API_KEY",
```

Comment:
@fvictorio did we decide this should become camel case?

Reply:
I think it should be camel case, as we are installing `eslint` with the recommended settings to our new users, and it enforces camel case.

Reply:
I switched the enums to camelcase so `bscTestnet` rather than `bsc_testnest`.